disnake/types/automod.py

```diff
@@ -118,6 +120,4 @@ class AutoModActionExecutionEvent(_AutoModActionExecutionEventOptional):
     rule_id: Snowflake
     rule_trigger_type: AutoModTriggerType
     user_id: Snowflake
-    content: str
     matched_keyword: Optional[str]
```

Comment:
heh, would it make sense to assume that this is not required as well?

Reply:
https://github.com/DisnakeDev/disnake/pull/530/commits/4a27eb9524d719e6802d1e44c2d95bb156f20027